packages/core/medusa-test-utils/src/module-test-runner.ts

```diff
@@ -43,8 +43,16 @@ export function moduleIntegrationTestRunner<TService = any>({
 
   process.env.LOG_LEVEL = "error"
 
-  moduleModels ??= Object.values(require(`${process.cwd()}/src/models`))
-  moduleModels = toMikroOrmEntities(moduleModels)
+  try {
+    moduleModels ??= Object.values(require(`${process.cwd()}/src/models`))
```

Comment:
thought: this path should probably be dynamic depending on whether we are using the test runner within a module pkg or a medusa project.

For example, if I use this in my Medusa project I get:

I am getting this:

```
❯ jest -- src/modules/car/__tests__/index.ts
 FAIL  src/modules/car/__tests__/index.ts
  ● Test suite failed to run

    Cannot find module '/Users/sebastianrindom/Developer/sebs-test-store/src/models' from 'node_modules/medusa-test-utils/dist/module-test-runner.js'
```

Reply:
should be fixed, I ll generate a snap

Reply:
no sorry I ve been too fast, I ll get back to it

Reply:
Just tested with the following and it works

<img width="1035" alt="Screenshot 2024-09-09 at 18 39 21" src="https://github.com/user-attachments/assets/d1d4258f-3572-4d61-9a53-61464b74a930">

<img width="1003" alt="Screenshot 2024-09-09 at 18 39 46" src="https://github.com/user-attachments/assets/d806e1a1-3731-4ff7-abd7-11d33137c032">

Reply:
Ill fix the tests tomorrow